example.py

```diff
@@ -44,22 +44,41 @@ def simulate(car, time_step=0.0002, interval=1):
 
 
 if __name__ == "__main__":
-    argparser = argparse.ArgumentParser()
+    mode_defaults = {
+        "square": {
+            "amplitude": 0.03,
+            "frequency": 0.1
+        },
+        "sine": {
+            "amplitude": 0.3,
+            "frequency": 0.04
+        },
+        "triangle": {
+            "amplitude": 0.05,
+            "frequency": 1.8
+        },
+        "bump": {
+            "amplitude": 0.05,
+            "frequency": 1.8
+        }
+    }
+
+    argparser = argparse.ArgumentParser(usage='%(prog)s [options]\nWe have some defaults depending on --mode. ' + str(mode_defaults))
```

Comment:
- Remove [options]
- How to pretty print `mode_defaults` instead of `str(mode_defaults`?

Reply:
### Pretty print

```
pipenv install pyyaml
```

could do this like so

```python
    argparser = argparse.ArgumentParser(usage='%(prog)s [options]\nWe have some defaults depending on --mode. ' + yaml.dump(mode_defaults, default_flow_style=False))
```

then it formats like

```
pipenv run python example.py --help
usage: example.py [options]
We have some defaults depending on --mode. bump:
  amplitude: 0.05
  frequency: 1.8
sine:
  amplitude: 0.3
  frequency: 0.04
square:
  amplitude: 0.03
  frequency: 0.1
triangle:
  amplitude: 0.05
  frequency: 1.8
```